app/controllers/dashboards_controller.rb

```diff
@@ -3,7 +3,6 @@ class DashboardsController < BaseController
   before_action :set_policy_with_context, except: [:refresh_dashboard]
   before_action :set_dashboard
   before_action :set_defence_requests, only: [:show, :closed]
-  before_action :set_dashboard_count, only: [:show, :closed]
```

Comment:
Right, that is not the issue I was referring to. We should have one single point that calls `policy_scope` with the user. From then on, we should only work with that whitelisted set of records - since they should always include every record we need. Does that make sense?